lib/process-services/process-list/components/process-list.component.ts

```diff
@@ -95,10 +86,6 @@ export class ProcessInstanceListComponent implements OnChanges, AfterContentInit
     @Input()
     size: number = PaginationComponent.DEFAULT_PAGINATION.maxItems;
 
-    /** Name of a custom schema to fetch from `app.config.json`. */
-    @Input()
-    presetColumn: string;
```

Comment:
if you remove an inout parameter is a BC

Reply:
Basically, we are not removing the property, it gets inherited from the parent.

Reply:
ok